tools/clang/lib/AST/DeclPrinter.cpp

```diff
@@ -852,6 +852,11 @@ void DeclPrinter::VisitStaticAssertDecl(StaticAssertDecl *D) {
 // C++ declarations
 //----------------------------------------------------------------------------
 void DeclPrinter::VisitNamespaceDecl(NamespaceDecl *D) {
+  // HLSL Change Begin - Don't emit built-in "vk" namesapce
```

Comment:
nit: typo "namesapce" -> "namespace"

Would you also be able to add a bit more detail about why we're not emitting this namespace in the comments here?

Reply:
I added a SPIR-V rewriter test to this PR, but I have difficulties actually running this test. Can you point me to the docs where the unit test procedure is explained? I can't find a `clangTest` project in my DXC VS solution.

The reason this namespace should not be emitted is because it's an internal declaration and DXC doesn't actually support parsing this declaration without additional arguments such as explicitly enabling template support and other HLSL 2021 features. It is also not necessary to print this declaration when rewriting HLSL source, because it is implicitly declared when we compile with the `-spirv` argument.

Reply:
I think it's `hcttest` in the HLSL console according to https://github.com/microsoft/DirectXShaderCompiler#running-tests, but I don't know for sure since I generally build and test on Linux which has a slightly different setup and just uses CMake's `ctest` command.

Also, could you put the explanation you wrote here in the comments in after line 855? Thanks!